cobspec/src/test/java/com/saralein/cobspec/controller/CoffeeControllerTest.java

```diff
@@ -0,0 +1,26 @@
+package com.saralein.cobspec.controller;
+
+import com.saralein.server.request.Request;
+import com.saralein.server.response.Header;
+import com.saralein.server.response.Response;
+import java.util.HashMap;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class CoffeeControllerTest {
+    @Test
+    public void returns418ResponseWithBody() {
+        Request request = new Request(new HashMap<String, String>(){{
+          put("method", "GET");
+          put("uri", "coffee");
+          put("version", "HTTP/1.1");
+        }});
+
+        Response response = new CoffeeController().createResponse(request);
+        Header header = response.getHeader();
+
+        assertEquals("HTTP/1.1 418 I'm a teapot\r\nContent-type: text/html\r\n\r\n", header.formatToString());
```

Comment:
is this a cob spec thing, is the coffee endpoint supposed to return 418 teapot?

Reply:
Yeah, it's a cobspec thing:

<img width="391" alt="screen shot 2017-12-19 at 6 53 18 pm" src="https://user-images.githubusercontent.com/8053416/34185926-05fe9716-e4ee-11e7-9e93-d0c7b3069328.png">